src/core/etl/src/Flow/ETL/DSL/functions.php

```diff
@@ -239,3 +239,8 @@ function sprintf(Expression $format, Expression ...$args) : Expression
 {
     return new Expression\Sprintf($format, ...$args);
 }
+
+function sanitize(Expression $expression, string $placeholder = '*', int $charactersLeft = 0) : Expression
```

Comment:
I would make `$placeholder` and `$charactersLeft` also an expression (so it can take lit or ref). 
Also I would rename `$charactersLeft = 0` into `$skipCharacters = 0`

Reply:
Done.